website/source/docs/builders/lxc.html.md

```diff
@@ -0,0 +1,82 @@
+---
+description: |
+    The `lxc` Packer builder builds containers for lxc1. The builder starts an LXC
+    container, runs provisioners within this container, then exports the container
+    as a tar.gz of the root file system.
+layout: docs
+page_title: LXC Builder
+...
+
+# LXC Builder
+
+Type: `lxc`
+
+The `lxc` Packer builder builds containers for lxc1. The builder starts an LXC
```

Comment:
This *should* work on most linux platforms such as Debian, Ubuntu, and Centos. I've only done it on Ubuntu with this code.
http://www.tecmint.com/install-create-run-lxc-linux-containers-on-centos/
Looks very promising for using this on Centos.

So, `lxc1` is what Ubuntu 15.10 and 16.04 call the original collection of lxc scripts. Other distros will probably just call it `lxc`.

From the Centos link Above: `# yum install lxc lxc-templates`

There is some confusion as `lxd` on Ubuntu 15.10 and 16.04 provide a reworked `lxc` command line written in go that has a web server daemon to move containers between hosts.

I'm not sure of the best way to distinguish between the two packages other than `lxc1`.